cli/src/commands/mod.rs

```diff
@@ -1,5 +1,29 @@
+use anyhow::{anyhow, Result};
+use reinfer_client::Client;
+
+use crate::utils;
+
 pub mod config;
 pub mod create;
 pub mod delete;
 pub mod get;
 pub mod update;
+
+pub fn warn_if_current_user_can_bypass_billing(client: &Client) -> Result<()> {
+    let current_user = client.get_current_user()?;
+
+    if current_user
+        .global_permissions
+        .contains(&reinfer_client::GlobalPermission::Debug)
+    {
+        println!(
+            "\n🚨⚠️ 👉 CAUTION 👈⚠️ 🚨\n\nThe operation you are about to perform will charge AI units\n"
+        );
+        let response = utils::read_from_stdin("To continue, type \"continue\"", None)?;
+
+        if response.to_lowercase() != "continue" {
+            return Err(anyhow!("Billable operation aborted by user"));
+        }
```

Comment:
If you want to make your life easier, I suggest adding `dialoguer` and dropping in `Confirm` here (you can give a prompt, then it handles all the retries/input parsing etc. for you): https://docs.rs/dialoguer/latest/dialoguer/struct.Confirm.html